### ISSUE TITLE
Unfocused selection is too light

### ISSUE THREAD

**Issue by SaschaNaz** [desktop/desktop#1669]
### Description

![github-scroll](https://cloud.githubusercontent.com/assets/3396686/26278724/498636e2-3ddc-11e7-8ffe-ed83b4d06f1e.gif)

### Version

**GitHub Desktop version:** 0.5.5

**OS version:** 10.0.15063

### Steps to Reproduce

1. Select a file on changes list
1. Scroll so that the file isn't visible anymore
1. Return

**Expected behaviour:** Selection should be kept

**Actual behaviour:** Selection is removed

**Reproduces how often:** 100%

**Comment by shiftkey** [comment]
I think this is a _focus_ issue rather than _selection_ as the menu item is a light gray instead of white when it scrolls back into view - and selecting the file doesn't change the diff.

**Comment by joshaber** [comment]
Yeah, I agree the selected, unfocused state is a bit too subtle. For comparison, Classic looks like:

![screen shot 2017-05-22 at 3 51 49 pm](https://cloud.githubusercontent.com/assets/13760/26325851/a7a198ae-3f06-11e7-8e65-15dfa1777f95.png)

**Comment by donokuda** [comment]
Just making sure, is the automatic deselection when scrolling away an expected behavior?

We can adjust the background gray, but I want to be sure that we're solving for the original problem reported here.

**Comment by niik** [comment]
> Just making sure, is the automatic deselection when scrolling away an expected behavior?

It's not but it's non-trivial to fix since those lists are virtualized. The DOM element which has focused gets destroyed when scrolling so Chromium unsets the active element. We'd need to apply some magic in our `List` implementation to keep track of focus globally and restore focus when appropriate.

> We can adjust the background gray, but I want to be sure that we're solving for the original problem reported here.

I played around with this the other day. I think $gray-200 is too dark (and also conflicts with our border colors which makes things look pretty terrible) but I agree that what we're using now ($gray-100) is a bit too much on the light side. I tried mixing the two to get something close to $gray-150 and ended up with `#ebeef1` (right side)

![image](https://user-images.githubusercontent.com/634063/28104863-5aaadfc0-66dd-11e7-825f-455387864a7c.png)
